doc/webui/user_attributes.rst

```diff
@@ -0,0 +1,30 @@
+.. _user_attributes:
+
+Additional user attributes
+--------------------------
+
+.. index:: User Attributes, Additional User Attributes
+
+Since version 3.6 privacyIDEA allows to manage additional internal attributes for
+users read from resolvers.
+These additional attributes are stored and managed within privacyIDEA.
+Administrators can manage attributes of users (see policies :ref:`admin_set_user_attributes`
+and :ref:`admin_delete_user_attributes`) and users can manage their attributes themselves
+(see policies :ref:`user_set_user_attributes` and :ref:`user_delete_user_attributes`).
+
+The additional attributes are added to the user object, whenever a user is used.
+The attributes are also added in the response of an authentication request. Thus these attributes
+could be used to pass additional attributes via the RADIUS protocol.
+
+The user attributes can also be used as additional conditions in policies
+(see :ref:`policy_conditions`) in the userinfo section.
+This way the additional attributes can be used to
+group users togeather within privacyIDEA and assign destinct policies to these groups,
+without the need to rely on information from the user store.
+
+The policy condition uses attributes (userinfo) from the user store and additional user
+attributes managed in privacyIDEA at the same time.
+
+.. note:: If the user already has a certain key in the userinfo that is fetched from the
+   resolver, the *additional user attributes* can also be used to *overwrite* the value
```

Comment:
You are right. But I am a bit uncertain how to solve this. 
Today I realize, it could be legitimate to have both, a "resolver" and a "user store". The "resolver" would be the logic that defines the connection to a "user store". 

An "attribute" in a "user store" would be something different than an "attribute" of a "resolver".
The attribute of a resolver could be the ``verify_tls`` of an LDAP-Resolver, while the attribute in the user store would be the ``email`` address of a user object.

I will probably rework this, I just realized, that it is not straigtforward.